src/main/kotlin/com/wisemencouncil/app/users/business/UserService.kt

```diff
@@ -17,4 +18,14 @@ class UserService @Autowired constructor(private val userRepository: UserReposit
 
         return mapEntityToUser(entity)
     }
+
+    @Transactional
+    fun createUser(signUpData: SignUpData): User? {
```

Comment:
Creo que `SignupData` es innecesario. Podemos tener simplemente `User`
```suggestion
    fun createUser(user: User): User? {
```